packages/hd/tests/key.test.ts

```diff
@@ -51,23 +50,17 @@ test("privateToPublic, derive public key from private key, HexString", (t) => {
 
 test("privateToPublic, derive public key from private key wrong length", (t) => {
   t.throws(() => {
-    privateToPublic(Buffer.from(""));
+    privateToPublic(Uint8Array.from([]));
   });
   t.throws(() => {
     privateToPublic(
-      Buffer.from(
-        "39d218506b30ca69b0f3112427877d983dd3cd2cabc742ab723e2964d98016",
-        "hex"
+      bytes.bytify(
+        "0x39d218506b30ca69b0f3112427877d983dd3cd2cabc742ab723e2964d98016"
       )
     );
   });
   t.throws(() => {
-    privateToPublic(
-      Buffer.from(
-        "0xbb39d218506b30ca69b0f3112427877d983dd3cd2cabc742ab723e2964d98016",
-        "hex"
-      )
-    );
+    privateToPublic(bytes.bytify("0x"));
```

Comment:
is there any reason to change the original hex to empty?

Reply:
In Node.js repl
```ts
> Buffer.from("0xbb39d218506b30ca69b0f3112427877d983dd3cd2cabc742ab723e2964d98016", "hex")
<Buffer >
```

Reply:
Or we can add one byte to the original hex.
`Old (32 bytes)`: `0xbb39d218506b30ca69b0f3112427877d983dd3cd2cabc742ab723e2964d98016`
`New (33 bytes)`: `0xbb39d218506b30ca69b0f3112427877d983dd3cd2cabc742ab723e2964d9801678`